### ISSUE TITLE
With NVDA running, interaction with Excel sheets in a workbook does not work correctly

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#10483]
### Steps to reproduce:
1. Launch NVDA.
2. Launch Excel 365 and open a workbook in it, which contains a few sheets.
3. When the workbook opens, press F6 to go to the navigation pane for the sheets.
4. Invoke the context menu. The context menu that opens is for a sheet in a workbook, so the focus is really in the tab strip for the sheets in the workbook. Press ESCAPE to close the context menu.
5. Use the left and right arrow keys to navigate between the tabs for the sheets in the workbook.

### Actual behavior:
Navigation between the tabs/sheets fails. Instead, at first press NVDA reports the currently selected cell in the sheet and on consecutive presses moves the selection between the cells as if the focus is in the sheet/table instead being in the tab strip for the sheets. Pressing F6 once more moves the focus to the next focusable element after the tab strip, which is the status bar, so the focus really still was in the tab strip.

### Expected behavior:
NVDA should not interfere as to how Excel works in this respect and report tab navigation as it does for other similar controls (tab strips). Check how Narrator does it, for example.

### System configuration
- Operating system: Windows 10 Pro version 1903 (build 18362.449), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-19146,d473de81, in bulgarian.
- Microsoft Excel version: Microsoft Excel for Office 365, 32-bit, MSO (16.0.12130.20232), click to launch, in bulgarian (information taken from the "About Excel" dialog).
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - 2019.2.1. The behavior is the same.

**Comment by Adriani90** [comment]
cc: @lukaszgo1, @leonardder maybe yet another focus issue here?

**Comment by Adriani90** [comment]
Using UIA for Excel solves this issue at least, but as we know using UIA in Excel still introduces other significant issues and alot of speech repetitions.
